### PR TITLE
Add some commands

### PR DESCRIPTION
**Pull request by gbittencourt**
Solve #1

### CODE REVIEW

**Comment by gbittencourt** [comment]
Try add more commands

File: README.md [hunk 1]
```diff
@@ -11,4 +11,5 @@ Este é o minicurso de Git do LKCAMP.
 * git status
 * git add
 * git commit
-...
+* git branch
+* git checkout
```
Comment (laispc) [inline README.md:15]: Really nice! This command is very useful!

File: README.md [hunk 2]
```diff
@@ -11,4 +11,5 @@ Este é o minicurso de Git do LKCAMP.
 * git status
 * git add
 * git commit
-...
+* git branch
```
Comment (laispc) [inline README.md:14]: :)

**Review by laispc — COMMENTED** [review]
Thanks for this PR! :)

**Pull request merged**